index.bs

```diff
@@ -3023,6 +3158,8 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     the [=attribution rate-limit cache=].
 1. [=list/Remove=] all [=attribution rate-limit records=] |entry| from the [=attribution rate-limit cache=] if the result of running
     [=can attribution rate-limit record be removed=] with |entry| and |source|'s [=attribution source/source time=] is true.
+1. [=Remove sources with incompatible attribution scope fields from cache=] with |source|.
```

Comment:
These steps should be performed before checking destination global rate limit (line 3130) to avoid privacy leak.